app/client/keybase/keybase.go

```diff
@@ -25,8 +25,8 @@ type Keybase interface {
 	DeriveChildFromKey(masterAddrHex, passphrase string, childIndex uint32) (crypto.KeyPair, error)
```

Comment:
Seeing how you introduced the `store_child` flag. Do we even need both `DeriveChild` and `StoreChild` functions anymore?

I feel like just having `DeriveChild` with `store_child` defaulting to `true` is simple, descriptive and maintains optionality.